gtd-timer/Extentions/ErrorHandlingMiddleware.cs

```diff
@@ -0,0 +1,63 @@
+﻿using Microsoft.AspNetCore.Http;
+
+using System;
+using System.Net;
+using System.Threading.Tasks;
+
+using Newtonsoft.Json;
+using gtdtimer.Extentions.Exceptions;
+
+namespace gtdtimer.Extentions
+{
+    public class ErrorHandlingMiddleware
+    {
+        private readonly RequestDelegate next;
+
+        public ErrorHandlingMiddleware(){}
```

Comment:
btw, is this for unit test only?

Reply:
Yes , I understand that is'nt the best way , So I will fix it.